aibolit/patterns/many_primary_ctors/many_primary_ctors.py

```diff
@@ -0,0 +1,43 @@
+import javalang.ast
+import javalang.parse
+import javalang.tree
+
+
+class ManyPrimaryCtors(object):
+    def value(self, filename: str):
+        tree = self.__file_to_ast(filename)
+
+        return self.__traverse_node(tree)
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
+        """
+        Takes path to java class file and returns AST Tree
+        :param filename:
+        :return: Tree
+        """
+
+        with open(filename) as file:
+            return javalang.parse.parse(file.read())
+
+    def __traverse_node(self, tree: javalang.ast.Node):
```

Comment:
@tribals Why not adding return type `-> []` to the method as you do in other methods ?

Reply:
See above.